cypress/e2e/practise.spec.ts

```diff
@@ -0,0 +1,61 @@
+import homePage from '../support/pages/home.page';
+import fileUpload from '../support/pages/file-upload.page';
+import webInputs from '../support/pages/web-inputs.page';
+import dragNDrop from '../support/pages/drag-n-drop.page';
+import actions from '../fixtures/actions-enums';
+
+describe('Automate different elements', () => {
+  beforeEach(() => {
+    cy.visit(Cypress.config().baseUrl!);
+  });
+
+  it('verify file upload by attaching file', () => {
+    homePage.search('upload');
+    homePage.selectCard(actions.FileUpload);
+    fileUpload.uploadFile('upload/pdf-test.pdf');
+    fileUpload.verifyFileUploaded('upload/pdf-test.pdf');
```

Comment:
Instead of hardcoding test data in the test cases we can use fixtures for PDF file path.